### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hoihrig**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hoihrig/terraform-modules/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # terraform-modules
 
-There are no secrets in this repository.
\ No newline at end of file
+There are no secrets in this repository.
+
```
Comment (hoihrig) [inline README.md:4]: why is this here !fix

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # terraform-modules
 
-There are no secrets in this repository.
\ No newline at end of file
+There are no secrets in this repository.
```
Comment (hoihrig) [inline README.md:3]: wat

File: README.md (hunk 1 above)
Comment (hoihrig) [inline README.md:4]: Oh yeah ... thats awesome !fixed
